components/application-operator/charts/application/templates/tests/application-gateway-tests.yaml

```diff
@@ -25,3 +25,13 @@ spec:
         - name: MOCK_SERVICE_PORT
           value: "8080"
   restartPolicy: Never
+---
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: {{.Release.Name }}-mtls-disable
```

Comment:
I do not think that this is a proper name as the `DestinationRule` is set for mock service, not for the `{{.Release.Name }}` itself.